packages/manager/apps/dedicated/client/app/dedicated/server/installation/ovh/dedicated-server-installation-ovh.html

```diff
@@ -604,15 +603,9 @@ <h4 class="panel-title">
                                 data-ng-class="{
                                     'deleting': partition == installation.partitionSchemeModels[setPartition.delModel]
                                 }"
+                                data-ng-class-odd="'oddRowColour'"
+                                data-ng-class-even="'evenRowColour'"
```

Comment:
**nitpick:** Adding an alternate coloured rows is not present in other places in Manager. This needs to be cleared from UX team.